src/Markdig/Parsers/FencedBlockParserBase.cs

```diff
@@ -1,231 +1,241 @@
+        /// </summary>
+        /// <param name="state">The parser processor.</param>
+        /// <param name="line">The being processed line.</param>
+        /// <param name="fenced">The fenced code block.</param>
+        /// <param name="openingCharacter">The opening character for the fenced code block (usually ` or ~)</param>
+        /// <returns><c>true</c> if parsing of the line is successfull; <c>false</c> otherwise</returns>
+        public delegate bool InfoParserDelegate(BlockProcessor state, ref StringSlice line, IFencedBlock fenced, char openingCharacter);
+
+
+        /// <summary>
+        /// Gets or sets the information parser.
+        /// </summary>
+        public InfoParserDelegate InfoParser { get; set; }
+
+        /// <summary>
+        /// A delegates that allows to process attached attributes
+        /// </summary>
+        public TryParseAttributesDelegate TryParseAttributes { get; set; }
+    }
+
+    /// <summary>
+    /// Base parser for fenced blocks (opened by 3 or more character delimiters on a first line, and closed by at least the same number of delimiters)
+    /// </summary>
+    /// <seealso cref="Markdig.Parsers.BlockParser" />
+    public abstract class FencedBlockParserBase<T> : FencedBlockParserBase where T : Block, IFencedBlock
+    {
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FencedBlockParserBase{T}"/> class.
+        /// </summary>
+        protected FencedBlockParserBase()
+        {
+            InfoParser = DefaultInfoParser;
+            MinimumMatchCount = 3;
+            MaximumMatchCount = Int32.MaxValue;
+        }
+
+        /// <summary>
+        /// Gets or sets the language prefix (default is "language-")
+        /// </summary>
+        public string InfoPrefix { get; set; }
+
+        public int MinimumMatchCount { get; set; }
+
+        public int MaximumMatchCount { get; set; }
+
+        /// <summary>
+        /// The default parser for the information after the fenced code block special characters (usually ` or ~)
+        /// </summary>
+        /// <param name="state">The parser processor.</param>
+        /// <param name="line">The line.</param>
+        /// <param name="fenced">The fenced code block.</param>
+        /// <returns><c>true</c> if parsing of the line is successfull; <c>false</c> otherwise</returns>
+        public static bool DefaultInfoParser(BlockProcessor state, ref StringSlice line, IFencedBlock fenced, char openingCharacter)
+        {
+            string infoString;
+            string argString = null;
+
+            // An info string cannot contain any backsticks (unless it is a tilde block)
+            int firstSpace = -1;
+            if (openingCharacter == '`')
+            {
+                for (int i = line.Start; i <= line.End; i++)
+                {
+                    char c = line.Text[i];
+                    if (c == '`')

```

Comment:
Shouldn't this using `openingCharacter ` instead here?

Reply:
This checks that there are no backtick characters in the info string if the opening character is also a backtick, as from 0.29 backticks in info strings are allowed for tilde fenced blocks.

Reply:
oops sorry, yes, let me check again the file (messy diff in github)